e2e/framework/config.go

```diff
@@ -46,6 +46,8 @@ type TestServerConfig struct {
 	ShowsLog                bool                 // Flag specifying if logs are shown
 	IsPos                   bool                 // Specifies the mechanism used for IBFT (PoA / PoS)
 	Signer                  *crypto.EIP155Signer // Signer used for transactions
+	MinValidatorCount       uint32
+	MaxValidatorCount       uint32
```

Comment:
Description is missing for both of these field, but present for the other 50

Reply:
fixed a91c866e0f6d742a0cc4af63f07196b4703ec96a

Reply:
Please add spacing 🙏

Reply:
Fixed 5a4af913579b3d51492802cfd59fb968b6865b8e